utils.py

```
@@ -0,0 +1,17 @@
+
+def validate_number(num):
+    # Basic validation without proper type checking
+    try:
+        float(num)
+        return True
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)